portal/home.php

```diff
@@ -50,7 +52,8 @@
     }
 }
 if ($newcnt > 0 && $_SESSION['portal_init']) {
-    $whereto = $_SESSION['whereto'] = '#secure-msgs-card';
+    $whereto = '#secure-msgs-card';
+    OpenEMR\Common\Session\SessionUtil::setSession('whereto', '#secure-msgs-card');
```

Comment:
this doesn't look right unless using this outside the portal for core openemr session. in portal can set sessions (there is no write block) and when use this setSession it will actually create a core (OpenEMR) session.

Reply:
@bradymiller unsure why I changed this. maybe a senior moment or I was tracking a bug. still, can't be right either way so thanks.

Reply:
yeah, this all started going back to when I converted the portal to a single page app plus, portal scripts are shared between just a portal session and an openemr session for dashboard.
Brady separated these out when he did the new session manager classes. You're right we had an issue with persisting where patient last viewed across refresh however, I fixed by adding a persist function on menu clicks. But session wasn't being saved correctly thus I added the sessionWrite() which made it work not realizing I was writing to an openemr session. Weird thing with php sessions(not cookies) the whereto session writes still appear in the $_SESSION super as if the same cookie!

Too much for my fragile brain!:)

In the end I had a bug in my persist function where the whereto was immediately being overwritten which caused me to embark on the sessionWrite changes in the first place.

So long story short, this is Brady's fault is what I'm going with.:)